sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -153,6 +173,16 @@ def _from_generated(cls, generated):
             )
         )
 
+    def __str__(self):
+        # type: () -> str
+        return "version={}, enabled={}, include_apis={}, retention_policy={}".format(
+            self.version, self.enabled, self.include_apis, self.retention_policy
+        )
+
+    def __repr__(self):
+        # type: () -> str
+        return "TableMetrics({})".format(self.__str__())
```

Comment:
Even though it's pretty safe to assume we'll never hit it for these models, we tend to add `[:1024]` to the end.